runtime/westend/src/lib.rs

```diff
@@ -1086,11 +1113,30 @@ pub type Executive = frame_executive::Executive<
 	frame_system::ChainContext<Runtime>,
 	Runtime,
 	AllPallets,
-	(),
+	StakingBagsListMigrationV8,
```

Comment:
TODO for some reason I am not seeing any logging for this when i run try-runtime